src/map/map.cpp

```diff
@@ -5102,6 +5101,37 @@ int mapgenerator_get_options(int argc, char** argv) {
 	return 1;
 }
 
+int map_data::getMapFlag(int flag) const {
+#ifdef DEBUG
```

Comment:
Shouldn't we display this for release mode as well?

Reply:
We could, but then this gets comparatively slower.
Previously, we just blindly accessed the array; we didn't check if it was out of bounds.